coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/LongRunningAction.java

```diff
@@ -47,10 +47,12 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.locks.ReentrantLock;
 
+import static jakarta.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+
 public class LongRunningAction extends BasicAction {
     private static final String LRA_TYPE = "/StateManager/BasicAction/LongRunningAction";
     private static final ScheduledExecutorService scheduler = new ScheduledThreadPoolExecutor(10);
-    private static final String DEACTIVATE_REASON = "deactivate";
+    public static final String DEACTIVATE_REASON = "deactivate failed";
```

Comment:
This is now in the full on public API of Narayana LRA - is this string wording something we want to commit to supporting long term?

Reply:
There are two reasons for why the message `String warn_saveState(String cause);` can be generated: it can be because of an exception or because the call to `BasicAction.deactivate()` returned `false` (and in this case, since I don't have the specific cause in hand, I report as it is).